### PR TITLE
Remove PHPUnit's at matcher from Export plugins tests

### PR DESCRIPTION
**Pull request by MauricioFauth**
Related to https://github.com/phpmyadmin/phpmyadmin/issues/16332.

### CODE REVIEW

File: test/classes/Plugins/Export/ExportXmlTest.php [hunk 1]
```diff
@@ -455,97 +457,30 @@ public function testExportData(): void
...
-            '<column name="fName3">NULL</column>',
-            $result
-        );
-
-        $this->assertStringContainsString(
-            '</table>',
-            $result
-        );
+        $this->assertStringContainsString('<!-- Table test_table -->', $result);
+        $this->assertStringContainsString('<table name="test_table">', $result);
```
Comment (williamdes) [inline test/classes/Plugins/Export/ExportXmlTest.php:474]: This could be a block?

**Review by williamdes — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Review by williamdes — APPROVED** [review]
Awesome work !

**Comment by williamdes** [comment]
https://github.com/phpmyadmin/phpmyadmin/pull/16707/checks?check_run_id=2035399993#step:8:30

We probably have a big problem, the step is marked as succeeded .. :/

**Comment by williamdes** [comment]
No worries about my previous comment, I will improve the CI steps descriptions titles

**Comment by mauriciofauth** [comment]
> We probably have a big problem, the step is marked as succeeded .. :/

On Travis, PHPUnit tests in random order was allowed to fail if I remember correctly.

**Comment by williamdes** [comment]
> > We probably have a big problem, the step is marked as succeeded .. :/
> 
> On Travis, PHPUnit tests in random order was allowed to fail if I remember correctly.

Yup
I got messed up, now all this is better with 328cdb6980b62bad361c04416584b022a3dbf294

**Pull request merged**